src/CFITSIO.jl

```diff
@@ -570,8 +629,9 @@ function fits_write_key(
     f::FITSFile,
     keyname::String,
     value::Union{Real,String},
-    comment::String,
-)
+    comment::String)
```

Comment:
The goal of this was probably to allow adding more arguments without disrupting git blame, which is very valuable.  I'd leave it as it was before
```suggestion
    comment::String,
)
```

Reply:
I've fixed this in 031d41b